Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SlidableListItem/SlidableListItemPage.xaml

```diff
@@ -27,7 +28,8 @@
                                        IsRightSwipeEnabled="{Binding DataContext.IsRightSwipeEnabled.Value, ElementName=Root, Mode=OneWay}"
                                        IsLeftSwipeEnabled="{Binding DataContext.IsLeftSwipeEnabled.Value, ElementName=Root, Mode=OneWay}"
                                        LeftCommand="{x:Bind ToggleFavorite}"
-                                       RightCommandRequested="SlidableListItem_RightCommandActivated"
+                                       RightCommand="{Binding DeleteItem, ElementName=Page, Mode=OneWay}"
```

Comment:
I may be wrong but `x:Bind` has `data:Item` as `DataContext` so it can't find the `DeleteItem` command which is inside the page itself.